test/service_test.js

```diff
@@ -48,7 +50,7 @@ suite('service', () => {
         // Close the request to /log after some time elapses.
         setTimeout(async function() {
           // At this point we should have a logcat process.
-          let [ps] = await exec('ps -au');
+          let [ps] = await exec('ps au');
```

Comment:
Confirmed that `ps au` works on both Linux and Mac, while the dashed version only works on Linux.